docs/pages/access-controls/guides/headless-login.mdx

```diff
@@ -0,0 +1,236 @@
+---
+title: Headless Login (Preview)
+description: Headless Login
+---
+
+## Introduction
+
+<Admonition type="warning" title="Preview">
+  Headless Login is currently in Preview mode.
+</Admonition>
+
+Headless Login provides Teleport users with a secure way to authenticate with WebAuthn from a remote machine. This enables use cases which usually are not compatible with WebAuthn features, such as:
```

Comment:
I feel like we should specifically mention that this is useful in environments where a browser is not available.

Reply:
Yeah, that works.